.github/workflows/tests-unit.yaml

```diff
@@ -35,6 +35,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r dev-requirements.txt
           pip install -r requirements.txt
+          pip install allure-pytest
```

Comment:
Same as above, would just like to add this to the dev requirements so it doesn't get forgotten about in the future.